website/content/en/docs/kubebuilder/operator-scope.md

```diff
@@ -0,0 +1,299 @@
+
+### Configuring Namespace scoped operators
+
+- Add a helper function in the `main.go` file:
+
+```go
+// getWatchNamespace returns the Namespace the operator should be watching for changes
+func getWatchNamespace() (string, error) {
+    // WatchNamespaceEnvVar is the constant for env variable WATCH_NAMESPACE
+    // which is the Namespace where the watch activity happens.
+    // this value is empty if the operator is running with clusterScope.
+    var watchNamespaceEnvVar = "WATCH_NAMESPACE"
+    
+    ns, found := os.LookupEnv(watchNamespaceEnvVar)
+    if !found {
+        return "", fmt.Errorf("%s must be set", watchNamespaceEnvVar)
+    }
+    return ns, nil
+}
+```
+
+- Use the environment variable value: 
+
+```go
+...
+watchNamespace, err := getWatchNamespace()
+if err != nil {
+    setupLog.Error(err, "unable to get WatchNamespace, " +
+        "the manager will watch and manage resources in all cluster")
+}
+
+mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+    Scheme:             scheme,
+    MetricsBindAddress: metricsAddr,
+    Port:               9443,
+    LeaderElection:     enableLeaderElection,
+    LeaderElectionID:   "f1c5ece8.example.com",
+    Namespace:          watchNamespace, // namespaced-scope when the value is not an empty string 
+})
+...
+```
+
+- Define the environment variable in the `config/manager/manager.yaml`:
+
+```yaml
+spec:
+  containers:
+  - command:
+    - /manager
+    args:
+    - --enable-leader-election
+    image: controller:latest
+    name: manager
+    resources:
+      limits:
+        cpu: 100m
+        memory: 30Mi
+      requests:
+        cpu: 100m
+        memory: 20Mi
+    env:
+      - name: WATCH_NAMESPACE
+        valueFrom:
+          fieldRef:
+            fieldPath: metadata.namespace 
+  terminationGracePeriodSeconds: 10
+``` 
+
+**NOTE** The above will set as value of `WATCH_NAMESPACE` the namespace where the operator will be deployed. 
+
+### Configuring cluster-scoped operators with MultiNamespacedCacheBuilder
+
+- Add a helper function in the `main.go` file :
+
+```go
+// getWatchNamespace returns the namespace the operator should be watching for changes
+func getWatchNamespace() (string, error) {
+    // WatchNamespaceEnvVar is the constant for env variable WATCH_NAMESPACE
+    // which is the namespace where the watch activity happens.
+    // this value is empty if the operator is running with clusterScope.
+    var watchNamespaceEnvVar = "WATCH_NAMESPACE"
+	
+    ns, found := os.LookupEnv(watchNamespaceEnvVar)
+    if !found {
+        return "", fmt.Errorf("%s must be set", watchNamespaceEnvVar)
+    }
+    return ns, nil
+}
+```
+
+- Use the environment variable value and check if it is an multi-namespace scenario:
+
+```go
+    ...
+watchNamespace, err := getWatchNamespace()
+if err != nil {
+    setupLog.Error(err, "unable to get WatchNamespace, " +
+        "the manager will watch and manage resources in all cluster")
```

Comment:
We're pretty much repeating the helper and main.go almost exactly as it is from the previous section so maybe remove that and just reference that above and just show `options.NewCache = cache.MultiNamespacedCacheBuilder`.

Or if not then make the same fixes to the helper code as I've pointed above.